packages/uilib/src/lib/plugins/type-designer/canvas.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
```

Comment:
Calling this component `Canvas` is misleading, it does not contain any Canvas, can we find a better name?

Reply:
I opted for ElementsContainer.